assets/src/components/button/style.scss

```diff
@@ -0,0 +1,83 @@
+/**
+ * Button styles,
+ */
+
+
+.newspack-button {
+	.components-button {
+	    padding: 0 20px 1px;
+		text-align: center;
+
+		font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", sans-serif;
+		font-size: 14px;
+		line-height: 18px;
+		font-weight: 500;
+		color: $muriel-gray-700;
+		text-shadow: none;
+
+		&.is-button {
+			background-color: $muriel-white;
+			border: none;
+			border-radius: 3px;
+			cursor: pointer;
+			height: 40px;
+			max-width: 360px;
+			max-height: 40px;
+			margin-top: 13px;
+			margin-bottom: 9px;
+
+			&:hover {
+				border-color: $muriel-gray-500;
+			}
+
+			&:active:enabled {
+				border-width: 2px 1px 1px;
+			}
+		}
+
+		&.is-primary {
+			background-color: $muriel-hot-pink-500;
+			border-color: #992053;
+			box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
```

Comment:
Any reason the `box-shadow` is in the `is-primary` block vs. `is-button`? I'm not certain, but would guess that the shadow should be the same for primary and other buttons?

Reply:
There's not really a shadow on the others - like `isDefault` - but there is a strong bottom border instead.